augmentation/gpt3_question_generator/gpt_generator.py

```diff
@@ -0,0 +1,35 @@
+import openai
+from .gpt import GPT, Example
+from .models import AugmentationRequest
+
+
+class GPT3QuestionGenerator:
+
+    def __init__(self, request_data: AugmentationRequest):
+
+        openai.api_key = request_data.api_key
```

Comment:
openai is global. what is the behaviour when 2 concurrent users hit the api? does the key changes per request or for the all users?